source/util.js

```diff
@@ -63,3 +65,16 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+function splitFileNameList(stdout) {
+	if (typeof stdout !== 'string') {
+		throw new TypeError('expected parameter stdout of type string');
+	}
+
+	return stdout.trim().split('\n').map(row => row.replace(/[|+-]/g, '').trim());
+}
+
+exports.getNewFilesIgnoredByNpm = async pkg => {
+	const ListNewFiles = splitFileNameList(await gitUtil.newFilesSinceLastRelease());
```

Comment:
The constant's name should be in camelCase 🐫

Reply:
fixed